_posts/2022-02-14-changelog-165.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #165"
+date: 2022-02-14 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8473] Provide support for 2022.1 EAPs
```

Comment:
@vlad20012 suggested a good idea - let's provide a bit more about EAP. For example, something like this:

```suggestion
* [#8473] Provide support for 2022.1 [EAP](https://www.jetbrains.com/resources/eap/) builds.
  Note, The Early Access Program gives you free access to pre-release versions of JetBrains software and doesn't require a paid IDE subscription
```